src/index.ts

```diff
@@ -40,12 +40,22 @@ const onRight: T.Task<void> = pipe(
  */
 export const exit: (program: TE.TaskEither<string, void>) => T.Task<void> = TE.fold(onLeft, () => onRight)
 
+/**
+ * @category utils
+ * @since 0.6.7
+ */
+export const compilerOptions: ast.ProjectOptions['compilerOptions'] = {
+  strict: true
+}
+
 const capabilities: Core.Capabilities = {
   example: Example,
   fileSystem: FileSystem,
   logger: Logger,
   ast: {
-    project: new ast.Project(),
+    project: new ast.Project({
+      compilerOptions
```

Comment:
Another way would be to read the compiler options from the user's `tsconfig.json`.
I think generally using `strict : true` is a good practice. But at the moment, I see no reason why a library shouldn't be able to make use of less strict options.

Reply:
> Another way would be to read the compiler options from the user's tsconfig.json

Good idea (maybe in another PR), I won't export `compilerOption` for the moment (by tagging it as internal).

> I see no reason why a library shouldn't be able to make use of less strict options

Well, since this is a documentation tool for libraries in the `fp-ts` ecosystem, I'd say that `strict: true` is a basic requirement :)